lib/src/dom/anchors.dart

```diff
@@ -4,6 +4,9 @@
 
 // Generated from Web IDL definitions.
 
+@JS()
+library anchors;
```

Comment:
no name needed! just `library;` is great!

Reply:
Done!